cmake/external_gtest.cmake

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 2.8.2)
+cmake_minimum_required(VERSION 3.0.0)
 
 project(googletest-download NONE)
 
 include(ExternalProject)
 ExternalProject_Add(googletest
   GIT_REPOSITORY    https://github.com/google/googletest.git
-  GIT_TAG           master
+  GIT_TAG           main
```

Comment:
maybe it would be better to depend on actual fixed tag to prevent future surprises (api breakage etc)

Reply:
(I hope they do have version tags, right?)

Reply:
And give up on [Live at Head](https://abseil.io/about/philosophy#upgrade-support) philosophy? :(

Reply:
all right :D